docs/tutorial/mojave-dark-mode-guide.md

```diff
@@ -24,8 +24,9 @@ to set the `NSRequiresAquaSystemAppearance` key in your app's `Info.plist` file
 </plist>
 ```
 
-If you are using [`electron-packager` >= 12.2.0](https://github.com/electron-userland/electron-packager) or
-[`electron-forge` >= 6](https://github.com/electron-userland/electron-forge) you can set the
+If you are using [`electron-packager` >= 12.2.0](https://github.com/electron-userland/electron-packager),
+[`electron-forge` >= 6](https://github.com/electron-userland/electron-forge) or
+[`electron-builder` >= 20.37.0](https://github.com/electron-userland/electron-builder) you can set the
 [`darwinDarkModeSupport`](https://github.com/electron-userland/electron-packager/blob/master/docs/api.md#darwindarkmodesupport)
```

Comment:
My comment was more about the fact that the builder option is not called `darwinDarkModeSupport`.  It should be in it's own paragraph below and reference the builder option name and docs.